lib/ex_unit/lib/ex_unit/callbacks.ex

```diff
@@ -234,28 +234,33 @@ defmodule ExUnit.Callbacks do
   end
 
   @doc false
-  def __merge__(_mod, context, :ok) do
+  def __merge__(mod, context, value) do
+    __merge__(mod, context, value, value)
+  end
+
+  @doc false
+  defp __merge__(_mod, context, :ok, _original_value) do
```

Comment:
Let's call this private function just `merge`, we don't need the `__` here :), they're usually used to mark functions that have to be public as for internal use. Also, don't worry about squashing commits if you want, we'll squash anyways when merging 👍.